config/csi-plugin/gkm-csi-plugin.yaml

```diff
@@ -0,0 +1,148 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-node
+  namespace: gkm-system
+spec:
+  selector:
+    matchLabels:
+      app: gkm-csi-node
+  template:
+    metadata:
+      labels:
+        app: gkm-csi-node
+        role: gkm-csi
+    spec:
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      #serviceAccount: gkm-csi-node-sa
+      nodeSelector:
+        hardware-type: gpu
+      tolerations:
+        - key: "gpu"
+          operator: "Equal"
+          value: "true"
+          effect: "NoSchedule"
```

Comment:
so for this longerterm we should do the patching though https://github.com/redhat-et/GKM/tree/main/config/kind-gpu in the same way we do for the agent - so that when we run on an actual node this doesn't block the csi-node-agent

Reply:
Fixed in #28.
Turns out the Agent wasn't working. It had `- key: "gpu"` and `hardware-type: gpu` in the yaml file as well. Also, the patch file needed to patch a `DaemonSet`, not a `Deployment`.  But this has been fixed and both the Agent and CSI patch the yaml to add the kind gpu selection.